docs/source/cli_help/status.rst

```diff
@@ -79,3 +79,5 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 -L, --list-commits-with-hashes    Additionally list the short hashes and messages of commits introduced on each branch.
 
 --no-detect-squash-merges         Only consider *strict* (fast-forward or 2-parent) merges, rather than rebase/squash merges, when detecting if a branch is merged into its upstream (parent).
+
+.. include:: status_indent.rst
```

Comment:
Hmmm does it need to be extracted to a separate file?

Reply:
Hmm, yeah, it doesn't

Reply:
It might be needed for the sake of #518... but let's keep it un-extracted/inline as for now